lib/intMap.js

```diff
@@ -0,0 +1,51 @@
+var hasOwn = Object.prototype.hasOwnProperty;
+
+/**
+ * A very simple polyfill for Map with limited functionality
+ * that only works for integer keys.
+ */
+function IntMap() {
+    this.map = {};
```

Comment:
Ah, gotcha. Agreed that it's not worth the hassle. Modern browsers and Nodes will just end up using `global.Map` anyway.